{"uuid":"d40e16e2-73f2-4cd2-96e5-b568dc792fba","name":"Scenario Outline: The user getting error message with invalid python code for Insertion page","children":["c957baad-33e9-459e-a9f6-8c8df4cabfd3"],"befores":[{"name":"AppHooks.ApplicationHooks.scenario(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1707870740531,"stop":1707870740531}],"afters":[{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1707870740673,"stop":1707870740673},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1707870740803,"stop":1707870740803},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1707870741210,"stop":1707870741211},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1707870741211,"stop":1707870741211},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1707870741514,"stop":1707870741514},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1707870741678,"stop":1707870741678},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1707870741712,"stop":1707870741712},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1707870741717,"stop":1707870741717}],"start":1707870740530,"stop":1707870741717}